src/OSPSuite.Infrastructure.Import/Core/Mappers/DataSetToDataRepositoryMapper.cs

```diff
@@ -97,12 +97,20 @@ private void convertParsedDataColumn(DataRepository dataRepository, KeyValuePair
          //loop over view rows to get the sorted values.
          foreach (var value in column.Value)
          {
-            if (value == null || double.IsNaN(value.Value)) //but we actually should not be allowing this at all right?
+            var adjustedValue =
```

Comment:
This is too hard to understand here. I have no idea what is going on

Reply:
@msevestre  Should make more sense now